canonical/canonical.com
# Departments overview build

## Done

- Built page as per [redesign](https://www.figma.com/file/KQSCmTqOjY4gfKvJPyvzj2/Canonical.com-latest-only?node-id=4563%3A8766)
- NOTE: #605 must be merged into feature branch first   

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-632.demos.haus/careers/departments/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Note: Department links do not work yet, they're coming in another PR

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5971
